images/ubuntu/19.04/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/ubuntu:19.04
+
+ENV NAME=ubuntu-toolbox VERSION=19.04
+LABEL com.github.debarshiray.toolbox="true" \
+      com.redhat.component="$NAME" \
+      name="$FGC/$NAME" \
+      version="$VERSION" \
+      usage="This image is meant to be used with the toolbox command" \
+      summary="Base image for creating Ubuntu 19.04 toolbox containers" \
+      maintainer="Debarshi Ray <rishi@fedoraproject.org>"
+
+COPY README.md /
+
+RUN apt-get update
+
+RUN apt-get -y upgrade
+
+COPY extra-packages /
+RUN apt-get -y install $(cat extra-packages | xargs)
```

Comment:
Does this not work:
```
RUN apt-get -y install $(<extra-packages)
```

Would be nice to avoid needless divergences in the Dockerfiles.

Reply:
Nope, it skips over the line because $(<extra-packages) ends up being empty. My guess is that there's a difference in the bash versions / behaviors.